### DISCUSSION TITLE
BRouter error message

### DISCUSSION THREAD

**Discussion by Basti32** [devemux86/cruiser#576] [Q&A] [answered]
Hallo
mit dem BRouter habe ich leider eine Fehlermeldung: Verfügbare Routingdaten prüfen.
Habe mir zwar die BRouter App heruntergeladen und auch Kartenmaterial dazu. Doch die Fehlermeldung bleibt.
Was kann ich tun, um diese Fehlermeldung zu beheben. 
Würde mich freuen, wenn ihr mir einige Tipps geben könntet.

Viele Grüße
Basti

Hello, 
unfortunately I have an error message with the BRouter: Check available routing data. I have downloaded the BRouter app and also the map material, but the error message remains. But the error message remains. What can I do to resolve this error message? I would be pleased if you could give me some tips.

Best regards 
Basti

**Comment by devemux86** [comment, marked as answer]
@Basti32 

You do not need the BRouter app with Cruiser. BRouter is included in Cruiser.
Use the menu "Routing | BRouter" to download your area.

Sie benötigen die BRouter-App nicht mit Cruiser. BRouter ist in Cruiser enthalten.
Verwenden Sie das Menü "Routing | BRouter", um Ihr Gebiet herunterzuladen.

**Comment by Basti32** [comment]
Hello @devemux86, 
thank you, it works now. 
best regards Basti
